mock/isbehandlerdialog/behandlerdialogMock.ts

```diff
@@ -102,3 +102,18 @@ export const behandlerdialogMock = {
 export const behandlerdialogMockEmpty = {
   conversations: {},
 };
+
+function base64ToArrayBuffer(base64: string) {
+  const binaryString = Buffer.from(base64, "base64").toString("utf8");
```

Comment:
Måtte bruke `Buffer.from` i stedet for `atob` for å dette til å fungere lokalt, ref https://www.designcise.com/web/tutorial/how-to-fix-atob-is-not-defined-error-in-node-js